helpdesk_project/tests/test_helpdesk_project.py

```diff
@@ -0,0 +1,54 @@
+#Ticket in a project
+'''
+from odoo.tests import common
+
+@classmethod
+class TestHelpdeskTicketProject(common)
+    def ticket_in_proyect(cls):
+        super(TestHelpdeskTicketProject, cls).setUpClass()
+        
+        Ticket = env['helpdesk.ticket']
+        Project = env["project.project"]
+        
+        #ticket belongs to a project
+        -we create the new ticket
+        cls.ticket = Ticket.create({
+            'name': 'Test',
+            'description': 'Ticket test',
+        })
+        
+        -assign the new ticket to a project
+        
+        cls.project = Project.create({
+            'name': 'Test Helpdesk-Project',
+        })
+        cls.ticket.write({
+            'project_id': cls.project.id,
+        })
+        
+        -button ticket count +1
+        cls.project.write({
+            'ticket_count': cls-ticket_count+1
+        })
+        -check the count of the button is right
+    
+    def test_helpdesk_ticket_project(self):
+        self.assertNotEquals(self.ticket.project, '/',
```

Comment:
when a ticket does not have a project established, the m2o field value is False, nor /, and the m2o field it should not be self.ticket.project... ¿_id?